docs/02-Scales.md

```diff
@@ -231,7 +231,7 @@ minUnit | String | 'millisecond' | The minimum display format to be used for a t
 
 When providing data for the time scale, Chart.js supports all of the formats that Moment.js accepts. See [Moment.js docs](http://momentjs.com/docs/#/parsing/) for details.
 
-#### Display Formats
+### Display Formats
```

Comment:
I don't think we will redeploy the doc until the next release (2.5) for which one we will use GitBook and thus all anchors should work correctly. Leave it like it is seems the best option :)